Mathlib/Analysis/SpecialFunctions/Trigonometric/LogDeriv.lean

```diff
@@ -0,0 +1,22 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+
+import Mathlib.Analysis.Calculus.LogDeriv
+import Mathlib.Analysis.SpecialFunctions.Trigonometric.Deriv
+
+/-!
+# Logarithmic Derivatives of Trigonometric Functions
+
+We compute the logarithmic derivatives of the sine and cosine functions.
+-/
+
+theorem logDeriv_sin : logDeriv (Complex.sin) = Complex.cot := by
```

Comment:
I think that it's OK to `import` the new `Calculus.LogDeriv` file in existing `SpecialFunctions.*` files and add `Complex.logDeriv_sin`, `Complex.logDeriv_cos`, `Complex.logDeriv_exp`, and similarly for `Real.*`.